spec/devise_saml_authenticatable/strategy_spec.rb

```diff
@@ -1,10 +1,10 @@
-require 'spec_helper'
+require 'rails_helper'
```

Comment:
This spec couldn't be run individually without including the rails dependencies.